Tests/RevenueCatUITests/CustomerCenter/CustomerCenterViewModelTests.swift

```diff
@@ -346,7 +380,8 @@ final class CustomerCenterViewModelTests: TestCase {
             // Should always show yearly subscription since it expires first
             expect(purchaseInformation.title) == yearlyProduct.title
             expect(purchaseInformation.durationTitle) == yearlyProduct.duration
-            expect(purchaseInformation.pricePaid) == .nonFree(formatPrice(yearlyProduct.price))
+
+            expect(purchaseInformation.pricePaid) == .nonFree(formatted(price: 29.99))
```

Comment:
should we test the `renewalPrice` here as well?

Reply:
same as https://github.com/RevenueCat/purchases-ios/pull/5069#discussion_r2089408695